src/styles.ts

```diff
@@ -32,18 +32,20 @@ export function addClassesForAllSizes(
 }
 
 export function attachAllStyles() {
-	// Create <style> element
-	const element = document.createElement('style');
-	element.type = 'text/css';
-	element.setAttribute('data-meta', prefix);
-	element.textContent = notAttachedCss;
-
-	// Add to the DOM
-	const container = document.querySelector('head');
-	if (!container) {
-		throw new Error('<head> tag not found');
+	if (typeof document !== 'undefined') {
```

Comment:
Would be nice to use an early return here instead of wrapping the whole thing in a condition.